pkgs/applications/networking/cluster/minikube/default.nix

```diff
@@ -13,15 +13,15 @@ buildGoModule rec {
   pname = "minikube";
   version = "1.15.1";
```

Comment:
Didn't the version changed?

Reply:
ahh shoot, you're right; looks like I didnt push up that change